tests/unit/benchmark_ipd.rst

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
```

Comment:
So this is an `rst` file which is the file extension for restructured text. Could you change this to be a `.py` file.